### DISCUSSION TITLE
Help to provide simple example, receiving and sendint TUSD

### DISCUSSION THREAD

**Discussion by gnomiik** [gochain/web3#270] [General]
Sorry for this question, but I am struggling to figure this out. How to receive TUSD via web3 client and how to send TUSD to another address via the web3 client. Thanks in advance!

**Comment by treeder** [comment]
To receive, you can create an account with: `web3 account create` . COPY AND SAVE THE PRIVATE KEY!

Then to send it, you'd be calling a contract so something like this:

```sh
export WEB3_PRIVATE_KEY=0xYOUR_PRIVATE_KEY
export WEB3_RPC_URL=INFURA_OR_SOME_RPC_URL
web3 contract call --address TUSD_CONTRACT_ADDRESS --abi erc20 --function transfer 0x_TO_ADDRESS AMOUNT_IN_WEI
```

**Comment by gnomiik** [comment]
Thank you for that! Almost have it together, but a couple of questions

1. What is INFURA_OR_SOME_RPC_URL ? Where do I find the URL to use?
2. What is the different between TUSD_CONTRACT_ADDRESS and 0x_TO_ADDRESS AMOUNT_IN_WEI ? I understand sending to the TUSD address, but don't know what the "IN_WEI" is.

Thank you for your time!!

**Reply by treeder** [reply]
1: You need an node provider or you can run your own node. Will depend on what blockchain you're using, but https://infura.io/ is one, https://www.alchemy.com/ is another. There are others too. 

2: https://www.investopedia.com/terms/w/wei.asp
